src/vishakha/FindPrimeNos.java

```diff
@@ -0,0 +1,29 @@
+/*Print all prime numbers between 2 to 100*/
+
+package vishakha;
+
+import java.util.Scanner;
+
+public class FindPrimeNos {
+	
+	void findPrimeNos(int startIndex, int endIndex){
+		for(int i=startIndex; i<=endIndex; i++){
+			int count = 0;
+			for(int innerIndex=1; innerIndex<=endIndex; innerIndex++){
```

Comment:
This is not correct !!
we should not start with 0 and for every number we dont want to run the loop till endIndex.
if number is 17, we should start loop from 2 and go till Math.sqrt(17).

Reply:
consider start to end range is from 1 to 2000.
when you process number 1000 is prime or not, you will divide it with 2 then 3 then 4 so on..
once it is divided with any number other than 1, you should break the loop (otherwise time complexity would be worst) and declare number is not prime.